tools/packaging/kernel/configs/fragments/gpu/nvidia.arm64.conf.in

```diff
@@ -0,0 +1,24 @@
+# Support for loading modules.
+# It is used to support loading GPU drivers.
+CONFIG_MODULES=y
```

Comment:
So the gpu driver should be inserted into kata image?

Reply:
Yes, so we can run unmodified containers, see here how it works: https://github.com/kata-containers/kata-containers/blob/main/docs/use-cases/NVIDIA-GPU-passthrough-and-Kata.md